src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -460,6 +460,19 @@ class ChooseGenerator : public Appendable {
     std::vector<int64_t> _weights;
 };
 
+// This is a wrapper generator. It wraps an int generator to generate strings.
+class StringIntGenerator : public Generator<std::string> {
```

Comment:
I'm torn on this versus having expressions just use format-string generator which has a lot more power. I'll leave it to you and Jim, though, since it's subjective and may make certain kinds of generator expressions a lot more concise.

Reply:
Yeah -- I go back and forth on this also. I used the format string generator for my actual use earlier, but it leads to stuff like this
`{^FormatString: {format: "%d", withArgs: [{^RandomInt:{min: 0, max: *NumItems}}]}}`
as opposed to 
`{^RandomInt:{min: 0, max: *NumItems}}}`
That example was in a ^Join node. There was also was a bit of surprise when I couldn't use an int as an argument to the ^Join generator. 

Anyways, up to you and @jimoleary whether to include this part or now.